lib/include/aws_tls.h

```diff
@@ -31,6 +31,20 @@
     #error "include FreeRTOS.h must appear in source files before include aws_tls.h"
 #endif
 
+/**
+ * @defgroup TlsErrors TLS Error Codes
+ * @brief Error codes returned by the TLS API.
+ *
+ * Note that TLS API may also propagate port-specific
+ * error codes, or codes from mbedTLS.
+ */
+/**@{ */
+#define TLS_ERROR_HANDSHAKE_FAILED    ( -2001 )   /*!< Error in handshake. */
```

Comment:
We should probably be explicit about the size and signedness of these.

Reply:
As discussed, leaving without explicit type to match error codes in other Amazon FreeRTOS files.